Sources/Misc/StoreKitVersion.swift

```diff
@@ -17,12 +17,13 @@ import Foundation
 @objc(RCStoreKitVersion)
 public enum StoreKitVersion: Int {
 
-    /// Always use StoreKit 1. StoreKit 2 will be used (if available in the current device) only for certain APIs
-    /// that provide a better implementation. For example: intro eligibility, determining if a receipt has
-    /// purchases, managing subscriptions.
+    /// Always use StoreKit 1. 
```

Comment:
The comment about StoreKit 2 was no longer true. We dont use StoreKit 2 at all if `.storeKit1` is selected.

Reply:
Oh that's right